bin/Powheg/runcmsgrid_powheg.sh

```diff
@@ -373,8 +373,8 @@ if [ -s pwg-stat.dat ]; then
   tail -${tail} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_tail
   head -${head} cmsgrid_final.lhe                           >  cmsgrid_final.lhe_F
   proclin=`expr $head + 1`
-  PROCESS=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
-  echo "  "$XSECTION"   "$XSECUNC"  1.00000000000E-00 "$PROCESS >>  cmsgrid_final.lhe_F
+  proc=`sed -n -e ${proclin}p  cmsgrid_final.lhe |  awk '{print $4}'`
```

Comment:
Hi @perrozzi , #1064 is just a synchronization copy from runcms_powheg.sh to runcmsgrid_powhegjhugen.sh and runcmsgrid_powheg_hdampgrid.sh. Maybe I shouldn't do the modification at all?

The problem is caused by the dash '-' not underscore '_'. For most cases, there are no dashes in the process name.